test/integration/local/payment-request/process-payment-request.test.js

```diff
@@ -96,19 +96,20 @@ describe('process payment requests', () => {
       }]
     })
 
-    expect(invoiceLinesRows[1].schemeCode).toBe('80001')
-    expect(invoiceLinesRows[1].accountCode).toBe('SOS273')
-    expect(invoiceLinesRows[1].fundCode).toBe('DRD10')
-    expect(invoiceLinesRows[1].agreementNumber).toBe('SIP00000000000001')
-    expect(invoiceLinesRows[1].description).toBe('G00 - Gross value of claim')
-    expect(parseFloat(invoiceLinesRows[1].value)).toBe(25000)
-
-    expect(invoiceLinesRows[0].schemeCode).toBe('80001')
-    expect(invoiceLinesRows[0].accountCode).toBe('SOS273')
-    expect(invoiceLinesRows[0].fundCode).toBe('DRD10')
-    expect(invoiceLinesRows[0].agreementNumber).toBe('SIP00000000000001')
-    expect(invoiceLinesRows[0].description).toBe('P02 - Over declaration penalty')
-    expect(parseFloat(invoiceLinesRows[0].value)).toBe(-10000)
+    const grossInvoiceLine = invoiceLinesRows.find(x => x.description === 'G00 - Gross value of claim' &&
+      x.value === 25000 &&
+      x.schemeCode === '80001' &&
+      x.accountCode === 'SOS273' &&
+      x.fundCode === 'DRD10')
+
+    const penaltyInvoiceLine = invoiceLinesRows.find(x => x.description === 'P02 - Over declaration penalty' &&
+      x.value === -10000 &&
+      x.schemeCode === '80001' &&
+      x.accountCode === 'SOS273' &&
+      x.fundCode === 'DRD10')
```

Comment:
Do these not need the agreementNumber anymore? Also is there anyway they could match the order of the mock invoice line defined  in the describe?

Reply:
Yes it does.  Good spot.  Added agreement and reordered as suggested